packages/charts/src/charts/line/lines.component.tsx

```diff
@@ -85,6 +85,7 @@ const Lines = ({
   } = useContext(ChartContext) as ChartContextType;
 
   const groupedMarks = useMemo(() => groupMarksByPosition(marks), [marks]);
+  const showAllMarks = stepMode || marks[0].radius <= lines[0].strokeWidth / 2;
```

Comment:
What if the data in chart will be empty - so the `marks` and `lines` do not have 0 index. I believe this use case will cause runtime error and unmount component from DOM.

Please extract this to separate function with fallback (could use `try {} catch`) - and write unit tests for the marks logic.